src/node_modules/app/elasticsearch.js

```diff
@@ -0,0 +1,123 @@
+export const descriptionsEN = {
+    avg: "Average",
+    weighted_avg: "Weighted Average",
+    cardinality: "Cardinality",
+    extended_stats: "Extended Statistics",
+    geo_bounds: "Geographic Bounds",
+    geo_centroid: "Geographic Centroid",
+    max: "Maximum",
+    min: "Minimum",
+    percentiles: "Percentiles",
+    percentile_ranks: "Percentile Ranks",
+    scripted_metric: "Scripted Metric",
+    stats: "Statistics",
+    string_stats: "String Statistics",
+    sum: "Sum",
+    top_tags: "Top Hits",
+    value_count: "Value Count",
+    median_absolute_deviation: "Median Absolute Deviation"
+}
+
+export const aggregationsPerType = {
+    ES_geo_point_obj: [
+        'cardinality', 
+        'geo_bounds', 
+        'geo_centroid', 
+        'value_count'
+    ],
+    ES_keyword: [
+        'value_count',
+        //'string_stats' //disabled until backend upgrade
+    ],
+    boolean: [
+        'cardinality', 
+        //'percentile_ranks', 
+        'value_count'
+    ],
+    integer: [
+        'cardinality', 
+        'extended_stats', 
+        //'percentile_ranks', 
+        'value_count'
+    ],
+    Date_YYYYMMDD_dash: [
+        'cardinality', 
+        'extended_stats', 
+        'percentiles', 
+        //'percentile_ranks', 
+        'value_count'
+    ],
+    date: [
+        'cardinality', 
+        'percentiles', 
+        //'percentile_ranks', 
+        'value_count'
+    ],
+    string: [
+        //'string_stats' 
+    ],
+    float: [
+        //'weighted_avg', 
+        'cardinality', 
+        'extended_stats', 
+        'percentiles', 
+        //'percentile_ranks', 
+        'value_count'
+    ],
+    'string[]': [
+        //'value_count'
+    ],
+    object: []
+}
+
+function defaultAggregationFactory (fieldName, fieldInfo) {
```

Comment:
`fieldInfo` here is unused, if is this for later use I'd add a comment, explaining why, or I'd remove it here and in this function calls